src/index.html

```diff
@@ -9,11 +9,17 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <title>Antisnake</title>
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
-<script type="text/javascript" src="./main.js"></script>
+  <div class="antisnake">
```

Comment:
use most general class names
```suggestion
  <div class="grid-container">
```